### ISSUE TITLE
GHA failures in aiidalab-launch

### ISSUE THREAD

**Issue by danielhollas** [aiidalab/aiidalab-launch#163]
Tests seem to be timing out randomly on GHA (e.g. after 30 minutes).
Interestingly, the pytest tests seem to always finish fine, but then the workflow step gets stuck? Very strange...

See for example

https://github.com/aiidalab/aiidalab-launch/actions/runs/3550589631/jobs/5972824480

**Comment by danielhollas** [comment]
Maybe duplicate of #154, see discussion there.

**Comment by unkcpz** [comment]
thanks for reporting this @danielhollas. When #154 happened, it is stuck in the command line rather than printing an error message and exiting so you can type new on the terminal, correct?

**Comment by danielhollas** [comment]
It is stuck and I needed to the Ctrl+C and only then the error was printed.

**Comment by unkcpz** [comment]
I see, I'll take a look at https://github.com/aiidalab/aiidalab-launch/issues/154. The error seems like the async function is called from another coroutine directly without `await`.

**Comment by danielhollas** [comment]
Yeah. Just note from the discussion there that Simon already spent some time on it and couldn't figure it out, so it might be a pain, maybe not the biggest priority now, although it is annoying. 

btw: Could you make a release of `aiidalab-launch`. I'd like to start using the two minor improvements that I implemented. Thanks!

**Comment by unkcpz** [comment]
Sure, I'll make the release. Thanks a lot for the fixes!

**Comment by danielhollas** [comment]
In failures [here](https://github.com/aiidalab/aiidalab-launch/actions/runs/4065691776/jobs/7000768652) and [here](https://github.com/aiidalab/aiidalab-launch/actions/runs/4065691776/jobs/7000768652)),the test suite no longer times out, but actually fails in a reasonable amount of time. The failures are too generic to pinpoint the issue, since it says the container fails to start. Probably more debug logging is needed to debug this, we need the pytest to somehow print the container logs when it fails to start.
